doc/articles/uno-development/splash-screen.md

```diff
@@ -0,0 +1,96 @@
+# Splash Screens
+
+Navigate to `YourProject.Shared > Assets` and add all [scales](#table-of-scales) of your spash screen image.
+
+## UWP
+
+In your UWP project head, navigate to `Package.appxmaifest > Visual Assets > Splash Screen` and add your images.
+
+ ![uwp-splash-screen](assets/uwp-splash-screen.JPG)
+
+## Android 
+
+1. In the Android project head, navigate to `Resources/values/Styles.xml`
+
+2. Add an `android:windowBackground` `item` to the `AppTheme` style.  
+```xaml
+<item name=�android:windowBackground�>@drawable/splash</item>
+```
+
+3. Navigate to `Resources/drawable`
+
+4. Create a `splash.xml` drawable  
+``` xaml
```

Comment:
```suggestion
``` xml
```

Reply:
In an earlier PR, Jerome recommended I use the xaml tag which is what I have been using. I am happy to change it, I just want to make sure I stay consistent

Reply:
It's good to use the `xaml` tag whenever it applies, but since this is a native Android resource it's not using Xaml, so plain `xml` makes more sense 😉